test/integration/simple_example/controls/client.rb

```diff
@@ -31,6 +32,19 @@
     its('exit_status') { should eq 0 }
   end
 
+  describe command('pip show forseti-security|grep Version') do
+    its('exit_status') { should eq 0 }
+    let(:forseti_version_formated) {
```

Comment:
We don't need these let helpers anymore; let's just compare `forseti_version` to `subject.stdout`.